continuous_integration.md

```diff
@@ -36,9 +36,9 @@ The current CI implementation is now available at the following URL:
 
     https://travis-ci.org/nest/nest-simulator
 
-Travis CI service has been integrated with the Github repository to automatically run the tests when code is pushed. Github integration is done by adding a simple YAML file to the project root.Travis results will appear in the Github pull requests and the primary log is visible in the Travis interface. 
+Travis CI service has been integrated with the Github repository to automatically run the tests when code is pushed. Github integration is done by adding a simple YAML file to the project root.Travis results will appear in the Github pull requests and the primary log is visible in the Travis interface.
 
-Build jobs
+## Build jobs
 
 The CI system is closely integrated to Github repository. Whenever some changes in the code
 are detected, the latest source code is downloaded to a worker machine which is running on Ubuntu 14.04 LTS(Trusty) and the following actions are performed:
```

Comment:
The list of steps that  follows should be an enumerated list, not preformatted code.